wordpress-mobile/WordPress-iOS
# [Gutenberg] Implement Capabilities changes from GutenbergDataSource

This PR implements changes from https://github.com/WordPress/gutenberg/pull/23956

To test:
- Run the project.
- Check that @-Mentions works as expected when it should be available.
- Check that Unsupported Block Editor works as expected when it should be available (i.e: Simple WPCom site).

PR submission checklist:
